### PR TITLE
Add check for dublicate options in config

### PR DESCRIPTION
**Pull request by EvgeniiMekhanik**
Check that admin can't set dublicate options
in config (max_miss, cookie_name and so on).

### CODE REVIEW

File: fw/http_sess_conf.c [hunk 1]
```diff
@@ -371,10 +380,22 @@ tfw_cfgop_cookie_options_set(TfwCfgSpec *cs, TfwCfgEntry *ce)
 
 	TFW_CFG_ENTRY_FOR_EACH_ATTR(ce, i, key, val) {
 		if (!strcasecmp(key, "Path")) {
+			if (was_path) {
+				T_ERR_NL("Duplicate argument: '%s'\n", key);
+				return -EINVAL;
+			}
```
Comment (keshonok) [inline fw/http_sess_conf.c:386]: This repetitive code pattern clearly asks for a macro that includes the check **and** the assignment to `true`. There are places where the assignment is done after additional piece of code such as parsing of the value. But if that parsing fails it won't matter anyway, so the assignment to `true` can be done earlier in the macro.

**Review by biathlon3 — APPROVED** [review]
LGTM

File: fw/http_sess_conf.c (hunk 1 above)
Comment (EvgeniiMekhanik) [reply fw/http_sess_conf.c]: Yes good point! Fixed

File: fw/cfg.h [hunk 2]
```diff
@@ -201,14 +201,21 @@ typedef struct {
...
-#define TFW_CFG_CHECK_VAL_N(op, req, spec, entry)				\
+#define TFW_CFG_CHECK_VAL_N(op, req, spec, entry)			\
 	if (! ((entry)->val_n op (req)) ) {				\
 		T_ERR_NL("%s: Invalid number of arguments: %zu, must "	\
 			 "be %s %d\n", (spec)->name, (entry)->val_n,	\
 			 #op, (req));					\
 		return -EINVAL;						\
 	}
 
+#define TFW_CFG_CHECK_VAL_DUBLICATE(name, val_was_set, code)		\
```
Comment (keshonok) [inline fw/cfg.h:212]: While spelling `dublicate` exists, it's very rare. `Duplicate` is the right word. Also, perhaps, the standard shortcut would suffice here for a shorter name? `TFW_CFG_CHECK_VAL_DUP`.

Comment (EvgeniiMekhanik) [reply fw/cfg.h]: Yes fixed

**Pull request merged**